packages/keycloak-user-management/src/components/forms/UserForm/index.tsx

```diff
@@ -103,7 +103,7 @@ const UserForm: React.FC<UserFormProps> = (props: UserFormProps) => {
   return (
     <Row className="layout-content">
       {/** If email is provided render edit user otherwise add user */}
-      <h5 className="mb-3">{props.initialValues.email ? 'Edit User' : 'Add User'}</h5>
+      <h5 className="mb-3">{props.initialValues.email ? 'Edit User' : 'User addition'}</h5>
```

Comment:
Should we stick to verb here instead of noun? As it was before?

Reply:
@dubdabasoduba

Reply:
Lets stick with `Add user`